src/Surfnet/StepupMiddleware/ApiBundle/Controller/DeprovisionController.php

```diff
@@ -57,6 +67,22 @@ public function deprovision(string $collabPersonId): JsonResponse
     public function dryRun(string $collabPersonId): JsonResponse
     {
         $this->denyAccessUnlessGrantedOneOff(['ROLE_DEPROVISION']);
+
+        $nameId = new NameId($collabPersonId);
+        $identity = $this->identityRepository->findOneByNameId($nameId);
+
+        if ($identity === null) {
+            throw new RuntimeException('Cannot forget an identity that does not exist.');
```

Comment:
Since this is a controller, might be better to throw `HttpInvalidParamException`?

Reply:
Moved the logic to the service and will ultimately result in a 200 for a dry-run